source/main.d

```diff
@@ -61,6 +61,10 @@ public struct ContainerCLI
     @Option("d", "directory", "Directory to find a root filesystem")
     string rootfsDir = null;
 
+    /** Immediately start at this directory in the container (cwd) */
+    @Option("workdir", null, "Start at this working directory in the container")
```

Comment:
Nit: Add "(defaults to '/')" here for completeness?

Reply:
Once that's fixed this can be merged